src/index.ts

```diff
@@ -1,4 +1,4 @@
-import { parse } from '@babel/babylon'
```

Comment:
This should be `@babel/babylon` as per the babylon repo.
https://github.com/babel/babylon#babylon-has-been-moved-into-the-main-babel-mono-repo-as-babelparser